docs/ibexa_products/ibexa_commerce.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Ibexa Commerce
+description: Discover Ibexa Commerce and its essential features, capabilities and benefits.
+month_change: true
+---
+
+# Ibexa DXP Commerce edition product guide
+
+## What is [[= product_name_com =]]
+
+[[= product_name_com =]] is the most powerful edition offered by [[= product_name_base =]].
+
+It assists you in managing each aspect of your customers' journey by combining content management, customization, and commerce functions into a single, dedicated solution.
+
+[[= product_name_com =]] offers a streamlined, unified platform where you can personalize each component of the online shopping experience.
```

Comment:
```suggestion
[[= product_name_com =]] offers a streamlined, unified platform where you can personalize each aspect of the online shopping experience.
```
Preferential, but "component" ssounds material, while aspect can also include emotions and sensations.